techniques/html/H84.html

```diff
@@ -149,26 +146,26 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-      <h2>Related Techniques</h2>
+      <h2>関連テクニック</h2>
       <ul>
          <li><a href="../html/H32">H32: Providing submit buttons</a></li>
          <li><a href="../general/G80">G80: Providing a submit button to initiate a change of context</a></li>
       </ul>
    </section>
 <section id="tests">
-      <h2>Tests</h2>
+      <h2>検証</h2>
       <section class="procedure" id="procedure">
-         <h3>Procedure</h3>
-         <p>For each <code class="language-html">select</code> element/button element combination:</p>
+         <h3>手順</h3>
+         <p>それぞれの <code class="language-html">select</code> 要素/ボタンの要素の組み合わせに対して::</p>
```

Comment:
コロンが重複しているので一つ削除しました。
```suggestion
         <p>それぞれの <code class="language-html">select</code> 要素/ボタンの要素の組み合わせに対して:</p>
```